Source/Dafny/Verifier/Translator.cs

```diff
@@ -6575,7 +6575,9 @@ AND  Apply(f,h0,bxs) == Apply(f,h0,bxs)
           var cf = (ConstantField)f;
           if (cf.Rhs != null && RevealedInScope(cf)) {
             var etran = new ExpressionTranslator(this, predef, NewOneHeapExpr(f.tok));
-            sink.AddTopLevelDeclaration(ff.CreateDefinitionAxiom(etran.TrExpr(cf.Rhs)));
+            if (!Attributes.Contains(cf.Attributes, "opaque")) {
```

Comment:
The Translator has a method `IsOpaqueFunction`, which checks if a function is opaque. I suggest you define an analogous method for constants and use it here. Alternatively, change the current `IsOpaqueFunction` to have the name `IsOpaque` and take any member declaration.